vagrant/lib/machine.rb

```diff
@@ -48,12 +48,13 @@ def configure_web_vm (node, host: nil, ip: nil, php_version: nil)
   Mount.provision(node)
   bootstrap_sh(node, ['node', 'web'], { php_version: php_version })
   
+  # run vhosts.sh on every reload
   node.vm.provision :shell, run: 'always' do |conf|
     conf.name = "vhosts.sh"
     conf.inline = "vhosts.sh --no-reload --quiet"
   end
   
-  service(node, { start: ['redis', 'httpd', 'nginx'] })
+  service(node, { start: ['redis', 'httpd', 'nginx', 'varnish'] })
```

Comment:
Updating since t's been simplified and I got rid of the `service` config command in varnish. Everything is now using service levels to account for reloads, with an initial start in the provisioner script.